test/feedback_test.dart

```diff
@@ -33,8 +33,48 @@ void main() {
       await tester.pumpAndSettle();
 
       userInputFields = find.byKey(const Key('feedback_bottom_sheet'));
+      final activeDrawingColor = find.byWidgetPredicate((widget) {
+        if(!(widget is IconButton)) return false;
+        var selectButton = widget as IconButton;
+        return selectButton.color != null && selectButton.onPressed != null;
+      });
 
       expect(userInputFields, findsOneWidget);
+      expect(activeDrawingColor, findsNothing);
+    });
+
+    testWidgets(' can open feedback in drawing mode', (tester) async {
+      final widget = BetterFeedback(
+        defaultNavigate: false,
+        child: Builder(
+          builder: (context) {
+            return const MyTestApp();
+          },
+        ),
+      );
+
+      await tester.pumpWidget(widget);
+      await tester.pumpAndSettle();
+
+      // feedback is closed
+      var userInputFields = find.byKey(const Key('feedback_bottom_sheet'));
+
+      expect(userInputFields, findsNothing);
+
+      // open feedback
+      final openFeedbackButton = find.text('open feedback');
+      await tester.tap(openFeedbackButton);
+      await tester.pumpAndSettle();
+
+      userInputFields = find.byKey(const Key('feedback_bottom_sheet'));
+      final activeDrawingColor = find.byWidgetPredicate((widget) {
+        if(!(widget is IconButton)) return false;
```

Comment:
Can we extract this as a helper function. It seems like it is the same as the callback above.

Reply:
It is - can try to put it into a function.